tests/utests/betting/game_serialization_tests.cpp

```diff
@@ -0,0 +1,189 @@
+#include <boost/test/unit_test.hpp>
+
+#include <scorum/protocol/operations.hpp>
+#include <scorum/protocol/betting/game.hpp>
+#include <scorum/protocol/betting/market.hpp>
+#include <scorum/protocol/betting/wincase.hpp>
+#include <scorum/protocol/betting/wincase_serialization.hpp>
+#include <scorum/protocol/betting/game_serialization.hpp>
+
+#include <defines.hpp>
+#include <iostream>
+
+namespace {
+using namespace scorum;
+using namespace scorum::protocol;
+using namespace scorum::protocol::betting;
+
+struct game_serialization_test_fixture
+{
+    create_game_operation get_soccer_game_operation() const
+    {
+        create_game_operation op;
+        op.moderator = "moderator_name";
+        op.name = "game_name";
+        op.game = soccer_game{};
+        op.start = time_point_sec{ 1461605400 };
+        op.markets = { { market_kind::result,
+                         { { result_home{}, result_draw_away{} },
```

Comment:
Why do you duplicate wincases: initial - opposite. Is it really impossible to extract opposite wincase?
Maybe use over_under, yes_no instead